tests/loadtests/vkloadtests/InstancedSampleBase.cpp

```diff
@@ -164,9 +153,9 @@ InstancedSampleBase::InstancedSampleBase(VulkanContext& vkctx,
                 vk::FormatFeatureFlagBits::eSampledImageFilterLinear;
     }
     if (kTexture->generateMipmaps) {
-		neededFeatures |=  vk::FormatFeatureFlagBits::eBlitDst
-			    | vk::FormatFeatureFlagBits::eBlitSrc
-		       | vk::FormatFeatureFlagBits::eSampledImageFilterLinear;
+                neededFeatures |=  vk::FormatFeatureFlagBits::eBlitDst
+                            | vk::FormatFeatureFlagBits::eBlitSrc
+                       | vk::FormatFeatureFlagBits::eSampledImageFilterLinear;
```

Comment:
Wrong tab size used.

Also, since this PR is removing most trailing spaces, lines 187, 190, 194 & 196 are blank lines with spaces on them. Please remove those spaces.

Reply:
There are a lot of trailing blanks in the source files. It would be straightforward to run a script over all the C/C++ files looking for that, but not the purpose of this PR.

Reply:
Do you have a script to do that? I'd love to get rid of the trailing blanks.

Reply:
Not per se, but something like

> find * -type f | egrep '\.(cc|cpp|hpp|c|h)$' | xargs sed -i -e 's/ *$//g'

would probably do the trick. Unless you have some source files with spaces in their names where a more complex find expression would be used instead of egrep to filter them out.